src/workerd/api/hibernatable-web-socket.c++

```diff
@@ -157,12 +175,26 @@ kj::Promise<WorkerInterface::CustomEvent::Result>
     message.setWebsocketId(kj::mv(eventParameters.websocketId));
   }
 
-  return req.send().then([](auto resp) {
+  auto promise = req.send().then([](auto resp) {
     auto respResult = resp.getResult();
     return WorkerInterface::CustomEvent::Result {
       .outcome = respResult.getOutcome(),
     };
-  });
+  }).eagerlyEvaluate(nullptr);
+
+  KJ_IF_SOME(hm, manager) {
```

Comment:
Hmmm, I'm not sure we want to be setting a timeout on the rpc call, I would think that we only want to track duration of the JS execution... I'm also not confident we even _have_ a `HibernationManager` when we call `sendRpc()`.